voctogui/lib/audioleveldisplay.py

```diff
@@ -1,17 +1,29 @@
 import math
 import cairo
 
-from gi.repository import Gtk, GLib
+from lib.config import Config
+from gi.repository import Gtk, GLib, Gst
 
 
 class AudioLevelDisplay(Gtk.DrawingArea):
     """Displays a Level-Meter of another VideoDisplay into a GtkWidget"""
     __gtype_name__ = 'AudioLevelDisplay'
 
     def __init__(self):
-        self.levelrms = []
-        self.levelpeak = []
-        self.leveldecay = []
+        self.audiostreams = int(Config.get('mix', 'audiostreams'))
+        meters = Config.get('mainvideo', 'vumeter')
```

Comment:
.get_ without a default-value will raise an Exception if the option isn't there, so the check with .has_option can be skipped and the code simplified to something like:

```
if meters == "all":
  meters = …

if meter > …:
  raise RuntimeError(…)
```